packages/intent/src/commands/install/guidance.ts

```
@@ -1,9 +1,10 @@
 import { existsSync, mkdirSync, readFileSync, writeFileSync } from 'node:fs'
 import { dirname, join } from 'node:path'
 import { parse as parseYaml } from 'yaml'
-import { formatIntentCommand } from '../command-runner.js'
-import { formatSkillUse, parseSkillUse } from '../skill-use.js'
-import type { ScanResult, SkillEntry } from '../types.js'
+import { formatIntentCommand } from '../../shared/command-runner.js'
+import { isGeneratedMappingSkill } from '../../skills/categories.js'
+import type { ScanResult, SkillEntry } from '../../shared/types.js'
+import { formatSkillUse, parseSkillUse } from '../../skills/use.js'
```

ESLint

[error] `../../skills/use.js` import should occur before type import of `../../shared/types.js`

(import/order)